README.md

```diff
@@ -47,7 +47,7 @@ This provide an interactive environment for debugging the CI test.
 
 ## Debugging
 
-Logging goes through the [DebugLogger](yarn-project/foundation/src/log/debug.ts) module in Typescript. To see the log output, set a `DEBUG` environment variable to the name of the module you want to debug, to `aztec:*`, or to `*` to see all logs.
+Logging goes through the [DebugLogger](yarn-project/foundation/src/log/debug.ts) module in Typescript. `LOG_LEVEL` controls the default log level, and one can set alternate levels for specific modules, such as `debug; warn: module1, module2; error: module3`.
```

Comment:
any reason this isn't simply JSON?
[["debug"], ["warn", "module1", "module2"], ["error", "module3"]]

Reply:
I'm thinking that `LOG_LEVEL` will often be set manually, and JSON is annoying to type. But I'm happy to go with any other DSL.

Reply:
That's fair, all good with me